packages/expo-crypto/src/Crypto.ts

```diff
@@ -0,0 +1,47 @@
+import { UnavailabilityError } from 'expo-errors';
+import ExpoCrypto from './ExpoCrypto';
+
+import { Algorithm, Encoding, DigestOptions } from './Crypto.types';
+export * from './Crypto.types';
+
+function assertAlgorithm(algorithm: Algorithm): void {
+  if (!Object.values(Algorithm).includes(algorithm)) {
+    throw new TypeError(
+      `expo-crypto: Invalid algorithm provided. Expected one of: Algorithm.${Object.keys(
+        Algorithm
+      ).join(', Algorithm.')}`
+    );
+  }
+}
+
+function assertData(data: string): void {
+  if (data == null || typeof data !== 'string' || !data.length) {
```

Comment:
```suggestion
  if (typeof data !== 'string') {
```

typeof will catch null & undefined, and empty strings can be hashed too